wiki/Tournaments/DACH4K/2023/en.md

```diff
@@ -0,0 +1,382 @@
+- Hybrid
+  1. [PUP - DVP (BringoBrango) \[INSANE\]](https://osu.ppy.sh/beatmapsets/1313205#mania/2721490)
+  2. [YOUNHA - Oort Cloud (Gahyeonln) \[valiant\]](https://osu.ppy.sh/beatmapsets/1880372#mania/4125284)
+- Long Note
+  1. [Levi Niha - Eleven Eight (HowToPlayLN) \[Irrational\]](https://osu.ppy.sh/beatmapsets/1039719#mania/2818759)
+  2. [TUYU - Itsuka Otona ni Nareru to Ii ne. (Cut Ver.) (Lott) \[Info-Illiterated Stan + 1.05x\]](https://osu.ppy.sh/beatmapsets/1964134#mania/4072333)
+  3. [Eagle - Boomy and The Boost (stupud man) \[Tempo\]](https://osu.ppy.sh/beatmapsets/1239184#mania/2576356)
+- SV
+  1. [Airuei - Framework (H1Pur) \[Structures\]](https://osu.ppy.sh/beatmapsets/1465676#mania/3010240)
+  2. [ellis - don't say ur sorry (sickrate remix) (RuleBlazing) \[No Thanks\]](https://osu.ppy.sh/beatmapsets/1568823#mania/3203798)
+- Tiebreaker
+  1. **[Chroma - lost found melody (AHHHHHHHHHHHHHH) \[insane\]](https://osu.ppy.sh/beatmapsets/1287246#mania/2672609)**
+
+### Qualifier
+
+1. [Pixel Terror - Game Over (ERA Sirbeyy)\[Stage 1: Determination\]](https://osu.ppy.sh/beatmapsets/2019572#mania/4205778)
+2. [HandBraJeans - Servant x Ray Bands - May I Help You? | SxS  B.o.B Mashup （Maxim-Miau）\[Stage 2: Novelty\]](https://osu.ppy.sh/beatmapsets/2019535#mania/4205690)
+3. [RD-Sounds feat. Meramipop/nayuta - Shiten (ERA Imperial) \[Stage 3: Deification\]](https://osu.ppy.sh/beatmapsets/2009752#mania/4181579)
+4. [PLASTIC GIRL IN CLOSET - ERROR (2022 Remake Ver.) (cut) (\[GS\]Rose) \[Stage 4: Melancholy\]](https://osu.ppy.sh/beatmapsets/2019549#mania/4205712)
+5. [Slayyyter - Alone (EimitMaggi999) \[Stage 5: Moving on\]](https://osu.ppy.sh/beatmapsets/2019984#mania/4206681)
+
+## Match results
+
+### Grand Finals
+
+Friday, 11 August 2023:
+
+| Player 1 |  |  | Player 2 | Match link | VOD link |
+| --: | :-: | :-: | :-- | :-- | :-- |
+| **ERA Leo** ::{ flag=DE }:: | **7** | 0 | ::{ flag=DE }:: sorgenfresser | [#1](https://osu.ppy.sh/community/matches/109944160) | [#1](https://www.youtube.com/watch?v=hJE-qzsi8wA) |
+
+Monday, 14 August 2023:
+
+| Player 1 |  |  | Player 2 | Match link | VOD link |
+| --: | :-: | :-: | :-- | :-- | :-- |
+| TsukiyaWhiskers ::{ flag=DE }:: | 5 | **7** | ::{ flag=DE }:: **ERA Leo** | [#1](https://osu.ppy.sh/community/matches/109993788) | [#1](https://www.youtube.com/watch?v=N52jHBxb0FQ) |
+| **ERA Leo** ::{ flag=DE }:: | **7** | 3 | ::{ flag=DE }:: TsukiyaWhiskers | [#1](https://osu.ppy.sh/community/matches/109995258) | [#1](https://www.youtube.com/watch?v=N52jHBxb0FQ&t=5694s) |
+
+### Finals
+
+Detailed statistics for this round can be found [here](https://docs.google.com/spreadsheets/d/1k3-HHhDco5j05QM44TbciF8-AxRaJqP3cTAMbUoQme8/edit#gid=1958097703).
+
+Friday, 5 August 2023:
+
+| Player 1 |  |  | Player 2 | Match link | VOD link |
+| --: | :-: | :-: | :-- | :-- | :-- |
+| ERA Leo ::{ flag=DE }:: | 4 | **6** | ::{ flag=DE }:: **TsukiyaWhiskers** | [#1](https://osu.ppy.sh/community/matches/109836329) | - |
```

Comment:
leave these empty cells empty without a dash, like this

```suggestion
| ERA Leo ::{ flag=DE }:: | 4 | **6** | ::{ flag=DE }:: **TsukiyaWhiskers** | [#1](https://osu.ppy.sh/community/matches/109836329) |  |
```

Reply:
i will fixed these in one commit later